src/Navigation/Home/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import "./index.css";
+function Home(props, ref) {
+  return (
+    <>
+      <div ref={ref} className="home">
+        <div className="top-1">
+          <div className="inside-1">
+            <div className="logo"></div>
+            <h1 style={{ color: "#FF7632", fontSize: "48px" }}>PROFILE</h1>
+            <h5
+              style={{
+                color: "#D4D4D4",
+                fontSize: "18px"
+              }}
+            >
+              My Personal Info
+            </h5>
+          </div>
+          <div className="line"></div>
+          <div className="inside-2">
+            <p>
+              Keenly interested in the universe of front-end development and
+              user experience. Aspiring to become a front-end web developer to
+              learn, explore and provide great user experience on web-based
+              digital products and services.
+            </p>
+          </div>
+        </div>
+        <div className="top-2">
+          <div className="canvas">
+            <div className="inside-3"></div>
+            <div className="bob">
+              <button className="bob-inside">
+                <a
+                  href={"https://registry.jsonresume.org/ShilpiMaurya"}
+                  target={"_blank"}
+                  style={{
+                    color: "white"
+                  }}
+                >
+                  Download Resume
+                </a>
+              </button>
+            </div>
+          </div>
+          <div className="inside-4">
+            <h3
+              style={{
+                color: "#FF7632",
+                fontSize: "20px",
+                marginTop: "0px"
+              }}
+            >
+              Personal Info
+            </h3>
+            <p className="data">Name: Shilpi Maurya</p>
```

Comment:
We should use `resume.json `values here by importing the file. That is the point of using `resume.json`.
If we need to update the resume-app, we will just change the `resume.json` and the resume-app will update automatically, instead of replacing those values in multiple project files.